docs/src/domains/cdn/_index.md

```diff
@@ -5,179 +5,168 @@ weight: 3
+see your CDN provider's official documentation.
+
+## Disable the Platform.sh router cache
+ 
+When you use a CDN, the Platform.sh router [HTTP caching](../../define-routes/cache.md) becomes redundant.
+To disable it, change your cache configuration for the routes behind a CDN to the following:
+ 
 ```yaml {location=".platform/routes.yaml"}
 "https://{default}/":
-    type: upstream
-    upstream: "app:http"
-    cache:
-        # Disable the HTTP cache on this route. It's handled by the CDN instead.
-        enabled: false
+   type: upstream
+   upstream: "app:http"
+   cache:
+       # Disable the HTTP cache on this route. It's handled by the CDN instead.
+       enabled: false
 ```
-
-## Preventing direct access
-
-When using a CDN, you might not want users to access your Platform.sh origin directly.
-There are three ways to secure your origin:
-
-- **Password protected HTTP Authentication**: Restrict access to your project with the [HTTP access control](../../environments/http-access-control.md).
-- **Allowing and denying IP addresses**: If your CDN doesn't support adding headers to the request to origin, you can allow the IP addresses of your CDN.
-- **Client-authenticated TLS**.
-
-### Password protected HTTP authentication
-
-This approach applies the same username and password to your production and all development environments.
-You can have developers enter credentials through their browser,
-or override the access control setting for each child environment.
-
-To use password protected HTTP Authentication:
-
-1. Consult your CDN's documentation to ensure it supports that feature.
-2. Generate a password of sufficient strength.
-3. Password protect your project using [HTTP access control](../../environments/http-access-control.md).
-4. Share the password with your CDN provider.
-5. Make sure the CDN adds a header to authenticate correctly to your origin by adding a custom header to the origin request with the base64 encoded `username:password` you chose in step 2.
-   For example: `Aladdin:OpenSesame` would become `Authorization: Basic QWxhZGRpbjpPcGVuU2VzYW1l`.
-
-### Allowing and denying IP addresses
-
-This approach applies the same IP restrictions to your production and all development environments.
-To remove it from development environments, you need to disable it on each environment
-or else create a single child of the default environment where it is disabled
-and them make all development branches off of that environment.
-
-You also have to update your configuration when your CDN updates their IP addresses.
-
-To allow and deny IP addresses:
-
-1. Set up your CDN.
-2. Get the IP range for your CDN provider:
-
+ 
+## Prevent direct access to your Platform.sh server
+ 
+When you use a CDN, there are three ways to prevent direct access to your Platform.sh server:
+ 
+- HTTP basic authentication
+- Allowing and denying IP addresses
+- Client-authenticated TLS
```

Comment:
:question: With the table of contents on the right, is the list necessary? I don't think it hurts to have it, just wondering the benefit.